content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -227,12 +274,13 @@ aggregationRule:
   clusterRoleSelectors:
   - matchLabels:
       rbac.example.com/aggregate-to-monitoring: "true"
-rules: [] # Rules are automatically filled in by the controller manager.
+rules: [] # The control plane automatically fills in the rules
```

Comment:
I changed this so that someone who doesn't know about the kube-controller-manager doesn't have to stop reading and go and look that up. The way I see it, this change makes the documentation more useful for people whose clusters are working. Someone whose cluster doesn't work will have more to investigate if they want to find which part of the control plane is failed. I hope that if you are running a control plane that you care about, you already have other tooling that can give you a strong hint.

Does that trade off make sense?

Reply:
I'd still make the distinction that a controller fills these in, which means it is done asynchronously.